pkg/asset/manifests/cloudproviderconfig.go

```diff
@@ -105,11 +106,11 @@ func (cpc *CloudProviderConfig) Generate(dependencies asset.Parents) error {
 		if err != nil {
 			return err
 		}
-		// TODO: AlibabaCloud: Add 'VpcID','ZoneID','VswitchID'
+
 		alibabacloudConfig, err := alibabacloudmanifests.CloudConfig{
 			Global: alibabacloudmanifests.GlobalConfig{
-				AccessKeyID:     client.AccessKeyID,
-				AccessKeySecret: client.AccessKeySecret,
+				AccessKeyID:     base64.StdEncoding.EncodeToString([]byte(client.AccessKeyID)),
+				AccessKeySecret: base64.StdEncoding.EncodeToString([]byte(client.AccessKeySecret)),
```

Comment:
Secrets should not be stored in the cloud config. Consumers should create credentials requests rather than reading from the cloud config.

Reply:
Okay, I think this is my mistake, because the RAM Role has been bound to ECS, so there is no need to specify AccessKeyID and AccessKeySecret as credentials. So i will delete here. https://github.com/openshift/cloud-provider-alibaba-cloud/blob/master/docs/getting-started.md#install-alibaba-cloudprovider-support